package.json

```diff
@@ -30,17 +30,18 @@
     "ember-cli-htmlbars": "^2.0.1",
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
     "ember-cli-sass": "^6.2.0",
+    "ember-cli-what-input": "^1.0.0",
     "ember-decorators": "2.3.1",
     "ember-tooltips": "3.0.0",
-    "foundation-sites": "^6.3.1",
-    "mapbox-gl": "^0.48.0"
+    "foundation-sites": "^6.3.1"
   },
   "devDependencies": {
     "babel-eslint": "^8.0.0",
     "babel-plugin-htmlbars-inline-precompile": "0.2.4",
     "broccoli-asset-rev": "^2.4.5",
     "broccoli-clean-css": "~1.1.0",
     "ember-ajax": "^3.0.0",
+    "ember-anchor": "^1.0.2",
```

Comment:
Do you know if this will need to be installed separately from within consuming apps? 

Let me know if there are few things like this. We can add certain hooks that subsequently install other packages on the consumer end.